modules/40-define-functions/600-type-annotations/description.en.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+name: Type Annotations
+theory: |
+
+  **Type annotations** are an ability to specify parameter types and return values for functions in Python. This is not a mandatory requirement of the language, but it can help programmers in further development, improve the readability of the code and increase its reliability.
+
+  Let's look at a simple example of a function without type annotations:
+  
+  ```python
+  def concat(first, second):
+      return first + second
+  ```
+  
+  This function concatenates two strings into one, but it's not immediately obvious from looking at the code what types the arguments are and that the function operates on strings rather than adding, for example, two numbers.
+  
+  If you use this function in your code in the future, you may need to check the types of the arguments before passing them into the function, which increases the size of the code and makes it harder to understand.
```

Comment:
If we use it further in the code, we should check the types of arguments before passing them to the function. It increases the size of the code and makes it harder to understand.